### PR TITLE
Solution example

### PR DESCRIPTION
**Pull request by andybash**

### CODE REVIEW

File: src/test/java/Qa2HomeworkOne.java [hunk 1]
```diff
@@ -7,33 +7,50 @@
 import org.openqa.selenium.WebElement;
 import java.util.List;
 
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (DenisKosachov) [inline src/test/java/Qa2HomeworkOne.java:10]: wtf?

Comment (andybash) [reply src/test/java/Qa2HomeworkOne.java:10]: lol

File: src/test/java/Qa2HomeworkOne.java [hunk 2]
```diff
@@ -7,33 +7,50 @@
...
+    private final By COMMENT_BTN = By.xpath(".//img[@src = '/v5/img/icons/comment-v2.svg']");
+    private final By LOGO = By.xpath(".//img[@src = 'https://f.pmo.ee/logos/4133/7b1236dab95abca45083322781760e97.svg']");
+    private final By TOP_MENU_CLASS = By.className("menu-items--top");
 
-        System.setProperty("webdriver.chrome.driver", "c://chromedriver.exe");
-        WebDriver browserWindow = new ChromeDriver();
-        browserWindow.manage().window().maximize();
-        browserWindow.get("http://tvnet.lv");
-        WebDriverWait wait = new WebDriverWait(browserWindow, 10);
+    private WebDriver driver = null;
```
Comment (DenisKosachov) [inline src/test/java/Qa2HomeworkOne.java:22]: how does it work?

Comment (andybash) [reply src/test/java/Qa2HomeworkOne.java:22]: `@BeforeEach` assigns a new driver object before each test, so it is only `null` at the beginning - when the test class object is created.

**Pull request merged**